flow/templates/print_status_stack.jinja

```diff
@@ -0,0 +1,21 @@
+{% extends "print_status.jinja" %}
+{% block detailed %}
+{% set format_head = format_job_id  + '%s' + format_label %}
+{% set format_operation = ("%%1s %%-%ss %%3s  " | format(column_width_operation,)) %}
+{% for job in jobs %}
+{% if parameters %}
+{% set para_output = ns.format_parameters | format(*job['parameters'].values()) %}
+{% endif %}
+{{ format_job_id | format(job['job_id'],) }}{{para_output}}{{ '%s' | format(job['labels']|join(', ')) }}
+{% for key,value in job['operations'].items() %}
+{% if (alias[value['scheduler_status']] != 'U' or value['eligible']) or all_ops%}
+{{ format_operation | bold_font(value['eligible'],pretty) | format(value | get_operation_status(pretty),key,'['+alias[value['scheduler_status']]+']',) }}
+{% endif %}
+{% endfor %}
+{% endfor %}
+{% if pretty %}
```

Comment:
I propose to move the pretty logic out of the template and pass different symbols to the template instead.

Reply:
^ similar to what I proposed above. Perhaps there's a convenient way to combine these and just replace the symbols? I might be wrong - it doesn't look like there is a one-to-one mapping with the set above, so maybe I'm confused about what this is doing.

Reply:
I now premade the legend string in python and moved the if pretty in python and pass the proper legend dictionary to template.